File: api/src/main/java/net/blueshell/api/repository/CommitteeMemberRepository.java
package net.blueshell.api.repository;

import net.blueshell.api.base.BaseRepository;
import net.blueshell.api.model.CommitteeMember;
import org.springframework.stereotype.Repository;

@Repository
public interface CommitteeMemberRepository extends BaseRepository<CommitteeMember, Long> {
}

File: api/src/main/kotlin/net/blueshell/api/repository/committee/CommitteeMemberRepository.kt
package net.blueshell.api.repository.committee

import net.blueshell.api.repository.base.BaseRepository
import net.blueshell.api.model.committee.CommitteeMember
import org.springframework.stereotype.Repository

@Repository
interface CommitteeMemberRepository : BaseRepository<CommitteeMember, CommitteeMember.Id>
